processing/src/test/java/io/druid/query/filter/BetweenDimFilterTests.java

```diff
@@ -0,0 +1,37 @@
+package io.druid.query.filter;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Injector;
+import com.google.inject.Key;
+import io.druid.guice.GuiceInjectors;
+import io.druid.guice.annotations.Json;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class BetweenDimFilterTests
```

Comment:
Is it possible to test each of the return cases for isAlphaNumeric both true and false?